### DISCUSSION TITLE
SLIDE_SWITCH not working inside GROUP

### DISCUSSION THREAD

**Discussion by SashaCX75** [zepp-health/zeppos-forum#93] [Developer Feedback]
If I put a SLIDE_SWITCH inside a GROUP container it doesn't show up. Other elements in the same GROUP container are displayed. In the simulator, everything works correctly, but on the GTR 4 clock it does not work.

**Comment by MyoungXUE** [comment]
Hello, thank you for your feedback, this may be a bug, we will confirm as soon as possible.

**Comment by MyoungXUE** [comment]
@SashaCX75 Hello, we have fixed this issue, please upgrade the simulator and the real machine to the latest version, thank you again for your feedback.
